tests/api_tests/nodes/test_views.py

```diff
@@ -844,12 +844,12 @@ def setUp(self):
         self.user.save()
         self.basic_auth = (self.user.username, password)
         self.project = ProjectFactory(is_public=False, creator=self.user)
-        self.registration_project = RegistrationFactory(creator=self.user, project=self.project)
+        self.registration_project = RegistrationFactory(creator=self.user, project=self.project, no_context=True)
```

Comment:
Thank you for the catch, that's dead code from trying to circumvent V2 API url_for restrictions.